src/validation/validation.ts

```diff
@@ -124,8 +164,28 @@ const SerializedSignatureSchema = z
   .strict();
 
 // Node schemas
-
-const NodeSchema: z.ZodType<any> = z.lazy(() =>
+type Node =
```

Comment:
deduplicate this type

Reply:
This `Node` type is for the serialized Nodes. It is different from the `Node` type in operation.ts. Did you mean that?

Reply:
ah yes I meant that. ignore the comment in that case!